update.py

```diff
@@ -63,6 +63,7 @@ def check_notices(session, sessionData):
         content_div = bs.find_all(content, 'div', {'id': 'printableArea'})[0]
         notice['text'] = content_div.decode_contents(formatter='html')
         notice['time'] = cds[6].string
+        notice['uid'] = id_ + "_" + year
```

Comment:
As discussed in metakgp, let's append '_{MD5HASH_OF_ATTACHMENT_RAW}'. This will make us send another mail if the attachment changes :D

Reply:
I've pushed this "feature" and I hope this will be a "feature".😝